cli/tests/ha-tests/ha_test.go

```diff
@@ -0,0 +1,123 @@
+package ha_tests
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+
+	test_utils "github.com/timescale/tobs/cli/tests/test-utils"
+)
+
+var RELEASE_NAME = "tobs"
+var NAMESPACE = "default"
+var PATH_TO_TOBS = "./../../bin/tobs"
+var PATH_TO_CHART = "./../../../chart/"
+var PATH_TO_TEST_VALUES = "./../testdata/f1.yaml"
```

Comment:
can we make this name more descriptive than f1.yaml to make it clear that this is ha related?

Reply:
Updated this, From now all test `values.yaml` don't include ha config. Tests will use `--enable-prometheus-ha` flag to deploy HA.